src/controllers/PartyController.js

```diff
@@ -17,5 +17,24 @@ class PartyController {
       data: [newParty],
     });
   }
+
+  static getAParty(req, res) {
+    const { partyId } = req.params;
+    const party = PartyModel.find(partyR => (
+      partyR.id === Number(partyId)
+    ));
+    if (party) {
+      res.status(200).json({
+        status: 200,
+        data: party,
+      });
+    } else {
+      return res.status(400).json({
+        status: 400,
```

Comment:
Please check this site  `https://httpstatuses.com/` and update your status code accordingly

Reply:
Okay. I'll do just that. Thank you

Reply:
I've updated the status code.